pkg/controllers/user/rbac/namespace_handler.go

```diff
@@ -20,6 +21,7 @@ import (
 const (
 	projectNSGetClusterRoleNameFmt = "%v-namespaces-%v"
 	projectNSAnn                   = "authz.cluster.auth.io/project-namespaces"
+	rtbOwnerAnn                    = "authz.cluster.cattle.io/rtb-owner"
```

Comment:
Ann is short for annotation and this is never set as an annotation, it is set as a label.
Also, a constant already exists for it in handler_base.go: `rtbOwnerLabel`